apps/meteor/server/startup/migrations/v305.ts

```diff
@@ -0,0 +1,10 @@
+import { Permissions } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 305,
```

Comment:
```suggestion
	version: 305,
	name: 'Delete unused view-history permission'
```
I know this is a very straightforward one, but I think it'd be good to have a name for this migration

Reply:
Agreed, I'll make the adjustments